lib/read-commands.js

```diff
@@ -14,7 +14,7 @@ const readCommands = async (commandsPath, dirPath) => {
 	const paths = fs.readdirSync(dirPath);
 	const commands = [];
 
-	const promises = paths.map(async path => {
+	const promises = paths.filter(path => ['.js', '.tsx'].find(ext => path.endsWith(ext))).map(async path => {
```

Comment:
I think this can be simplified as just `path.endsWith('.js') || path.endsWith('.tsx')`.

Reply:
`.some()` would be more appropriate here instead of `find()`